test/testautomation_platform.c

```diff
@@ -386,7 +386,7 @@ int platform_testSetErrorEmptyInput(void *arg)
 int platform_testSetErrorInvalidInput(void *arg)
 {
     int result;
-    const char *invalidError = NULL;
+    const char *invalidError = "";
```

Comment:
We should do this in main SDL2 and SDL3 repositories (too)

Reply:
ok, i do that

Reply:
SDL2 and SDL3 has this change now, so I guess we can merge this